crates/trigger-http/src/lib.rs

```diff
@@ -82,7 +86,14 @@ impl<F: RuntimeFactors> Trigger<F> for HttpTrigger {
     type InstanceState = ();
 
     fn new(cli_args: Self::CliArgs, app: &spin_app::App) -> anyhow::Result<Self> {
-        Self::new(app, cli_args.address, cli_args.into_tls_config())
+        let find_free_port = cli_args.find_free_port;
+
+        Self::new(
+            app,
+            cli_args.address,
+            cli_args.into_tls_config(),
+            find_free_port,
```

Comment:
You can probably write `cli_args.find_free_port` - the local variable doesn't really add any explanatory power.

Reply:
that was my initial thoughts as well, but the issue is `cli_args.into_tls_config()` consumes `self`. this approach is a way to move `find_free_port` value into a new variable. 

The readability is agreeable not the best especially in reference to the pattern already in the code but that seems like the way out here unless I would have to do some adjustment to  `into_tls_config()`